challenge.md

```diff
@@ -19,6 +27,11 @@
       FullStack Web Development.
     </p>
   </section>
+  <script>
+    const subtitle = document.querySelector('h2');
```

Comment:
If possible, selecting elements using className is always the best option.

```suggestion
    const subtitle = document.querySelector('.subtitle');
```

Reply:
Yes but if we need to delete it, i thought to don't selected by the class

Reply:
@nilete22 
Sure, but removing class is next step. You can use class to select the element to ensure you are selecting the correct one. Other `h2` could exist in the nodes tree... then you probably will select the wrong one.

Using class name as selector is always safer than using the tag element itself.